### ISSUE TITLE
v-validate-class replaces children expressions

### ISSUE THREAD

**Issue by giohappy** [kazupon/vue-validator#252]
vue expressions are removed when wrapped inside a parent with a v-validate-class directive:

Here {{relfield.label}} becomes empty if v-validate-class is used

```
<div v-if="isVisible(relfield)" class="form-group has-feedback" v-validate-class>
  <label :for="relfield.name" class="col-sm-4 control-label">{{ relfield.label }}<span v-if="relfield.validate && relfield.validate.required">*</span></label>
  <div class="col-sm-8">
    <input :field="relfield.name" v-validate="relfield.validate" v-disabled="!isEditable(relfield)" class="form-control" v-model="relfield.value" :id="relfield.name" :placeholder="relfield.input.label">
  </div>
</div>
```

**Comment by giohappy** [comment]
I can see that v-validate-class replaces its child with a new fragment which, I suppose, doesn't compile expressions.
I've solved setting the class by hand:

```
<div v-if="isVisible(relfield)" class="form-group has-feedback" :class="[$validation[relfield.name].invalid ? 'invalid': '']">
  <label :for="relfield.name" class="col-sm-4 control-label">{{ relfield.label }}<span v-if="relfield.validate && relfield.validate.required">*</span></label>
  <div class="col-sm-8">
    <input :field="relfield.name" v-validate="relfield.validate" v-disabled="!isEditable(relfield)" class="form-control" v-model="relfield.value" :id="relfield.name" :placeholder="relfield.input.label">
  </div>
</div>
```

**Comment by marceloch2** [comment]
@giohappy Thanks a lot, saved me.

**Comment by kazupon** [comment]
released v3.0.0-alpha.1
https://github.com/vuejs/vue-validator/releases/tag/v3.0.0-alpha.1

In 3.0 later, we can use the `classes` props of `validity` component.
see the [example](https://github.com/vuejs/vue-validator/tree/dev/examples/classes)
